tasks/deploy.yaml

```diff
@@ -15,8 +15,8 @@ tasks:
 
   - name: k3d-istio-bundle
```

Comment:
nit: could change the name here, although I don't think this task is used anywhere?

Reply:
oof yeah okay. Funny thing is i changed the zarf package name in that task and still missed that.